robotidy/decorators.py

```diff
@@ -0,0 +1,25 @@
+class configurable:  # noqa
+    def __init__(self, fun):
+        self.fun = fun
+
+    def __set_name__(self, owner, name):
+        if not hasattr(owner, 'configurables'):
+            owner.configurables = set()
+        owner.configurables.add(name)
```

Comment:
Here we're 'registering' configurable name in transformer class - so it will raise exception if someone will try to configure not existing param:
```
robotidy  --transform MyTransform:i_dont_exist:5
ValueError("Invalid configurable name: ':i_dont_exist' for transformer: 'MyTransform'")
```